mica-rest/src/main/java/org/obiba/mica/person/rest/PersonsResource.java

```diff
@@ -37,24 +38,28 @@ public PersonsResource(Dtos dtos, PersonService personService, EventBus eventBus
 
   @GET
   @Path("/study/{studyId}")
+  @RequiresPermissions({ "/draft/individual-study:VIEW", "/draft/harmonization-study:VIEW", "/draft/network:VIEW" })
```

Comment:
You need to be more specific, with the studyId, because mica allows to have specific permissions on the draft documents.